sqlmesh/core/engine_adapter/shared.py

```diff
@@ -328,7 +328,7 @@ def internal_wrapper(*args: t.Any, **kwargs: t.Any) -> t.Any:
             container[key] = expression  # type: ignore
             if catalog_support.is_single_catalog_only:
                 if catalog_name != engine_adapter._default_catalog:
-                    logger.warning(
+                    raise SQLMeshError(
                         f"{engine_adapter.dialect} requires that all catalog operations be against a single catalog: {engine_adapter._default_catalog}. Ignoring catalog: {catalog_name}"
```

Comment:
Should we delete the "Ignoring catalog: ..." part?

Reply:
thanks updated